src/JLSOFile.jl

```diff
@@ -79,13 +105,24 @@ function Base.:(==)(a::JLSOFile, b::JLSOFile)
         a.version == b.version &&
         a.julia == b.julia &&
         a.image == b.image &&
-        a.pkgs == b.pkgs &&
+        a.manifest == b.manifest &&
         a.format == b.format &&
         a.compression == b.compression &&
         a.objects == b.objects
     )
 end
 
 Base.names(jlso::JLSOFile) = collect(keys(jlso.objects))
+function Base.getproperty(jlso::JLSOFile, attr::Symbol)
+    if attr === :pkgs
+        @warn "pkgs property is deprecated, use .manifest instead"
```

Comment:
```suggestion
        @Base.depwarn("pkgs property  is deprecated, use .manifest instead")
```

Reply:
Doesn't `depwarn` require that specify a symbol though?

Reply:
Yeah, that isn't valid syntax.